.github/workflows/pr.yml

```diff
@@ -0,0 +1,11 @@
+name: PR
+
+on:
+    pull_request:
+        branches: [ $default-branch ]
```

Comment:
@esensar does the $default-branch placeholder work?

Doesn't seem so according to https://stackoverflow.com/questions/64781462/github-actions-default-branch-variable

I can see the action is recognized but it can not be triggered.

Reply:
My bad, this is meant only for templates and I copied the template over instead of generating it using GitHub UI. I will update this in both this and the Launcher PR.

Reply:
Oh, you have done it. Nevermind :smile:

Reply:
Should I then proceed with adding these workflows to all the other apps, or would you like some time to pass to test this out?

Reply:
>would you like some time to pass to test this out?

Seems to be working fine 🎉

https://github.com/FossifyOrg/Gallery/pull/303/checks

> Should I then proceed with adding these workflows to all the other apps

That would be great, thanks!